packages/runtime/container-runtime/src/opLifecycle/duplicateBatchDetector.ts

```diff
@@ -75,4 +86,26 @@ export class DuplicateBatchDetector {
 			}
 		});
 	}
+
+	/**
+	 * Returns a snapshot of the state of the detector which can be included in a summary
+	 * and used to "rehydrate" this class when loading from a snapshot.
+	 *
+	 * @returns - A serializable object representing the state of the detector, or undefined if there is nothing to save.
+	 */
+	public getRecentBatchInfoForSummary(
+		telemetryContext?: ITelemetryContext,
+	): [number, string][] | undefined {
+		if (this.batchIdsBySeqNum.size === 0) {
+			return undefined;
+		}
+
+		telemetryContext?.set(
+			"fluid_DuplicateBatchDetector",
```

Comment:
```suggestion
			"fluid_DuplicateBatchDetector_",
```

This will create property like `"fluid_DuplicateBatchDetectorrecentBatchCount"`. Adding `_` to the end will result in `fluid_DuplicateBatchDetector_recentBatchCount` instead.